bot/exts/evergreen/githubinfo.py

```diff
@@ -120,7 +123,7 @@ async def github_repo_info(self, ctx: commands.Context, *repo: str) -> None:
             return
 
         async with ctx.typing():
-            repo_data = await self.fetch_data(f"{GITHUB_API_URL}/repos/{quote(repo)}")
+            repo_data = await self.fetch_data(f"{GITHUB_API_URL}/repos/{quote_plus(repo)}")
```

Comment:
This somehow broke the command. It fails to find the repo with `.gh repo python/cpython`, but that works fine on the main branch.

Reply:
Yeah, I should have only used it when we are looking up usernames, this piece of code right now encodes the `/` sign.